core/tests/webdriverio_utils/ExplorationPlayerPage.js

```diff
@@ -220,6 +220,13 @@ var ExplorationPlayerPage = function() {
     ).toMatch(richTextInstructions);
   };
 
+  this.waitForInvisibilityOfLessonCompletionMessage = async function() {
```

Comment:
Same here, maybe waitForLessonCompletionMessageToDisappear?

Reply:
Thanks @nikitaevg, I have made the suggested changes.